ml-agents/tests/yamato/scripts/run_llapi.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+from mlagents_envs.environment import UnityEnvironment
```

Comment:
This and the gym code were copied from the old notebooks, and cleaned up slightly to remove matplotlib display, conform to flake8, and close the environment in a `finally`.